package.json

```diff
@@ -8,7 +8,8 @@
     "@kiva/kv-tokens",
     "@kiva/kv-loan-filters",
     "@kiva/kv-shop",
-    "@kiva/kv-activity-feed"
+    "@kiva/kv-activity-feed",
+	"@kiva/vite-plugin-vue-lib-css"
```

Comment:
Strange that the package.json has an indentation difference.

Reply:
Whoops, mixed tabs and spaces